Content.Shared/Kitchen/RecipeManager.cs

```diff
@@ -38,7 +38,15 @@ public override int Compare(FoodRecipePrototype? x, FoodRecipePrototype? y)
 
                 var nx = x.IngredientCount();
                 var ny = y.IngredientCount();
-                return -nx.CompareTo(ny);
+                if (-nx.CompareTo(ny) != 0)
```

Comment:
Same as the above. It's also preferable to either keep the original line in the commented-out form, or to describe in a comment how the behavior was changed compared to upstream.